src/bleChannels.js

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2015 - 2017, Nordic Semiconductor ASA
+ *
+ * All rights reserved.
+ *
+ * Use in source and binary forms, redistribution in binary form only, with
+ * or without modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions in binary form, except as embedded into a Nordic
+ *    Semiconductor ASA integrated circuit in a product or a software update for
+ *    such product, must reproduce the above copyright notice, this list of
+ *    conditions and the following disclaimer in the documentation and/or other
+ *    materials provided with the distribution.
+ *
+ * 2. Neither the name of Nordic Semiconductor ASA nor the names of its
+ *    contributors may be used to endorse or promote products derived from this
+ *    software without specific prior written permission.
+ *
+ * 3. This software, with or without modification, must only be used with a Nordic
+ *    Semiconductor ASA integrated circuit.
+ *
+ * 4. Any software provided in binary form under this license must not be reverse
+ *    engineered, decompiled, modified and/or disassembled.
+ *
+ * THIS SOFTWARE IS PROVIDED BY NORDIC SEMICONDUCTOR ASA "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+ * MERCHANTABILITY, NONINFRINGEMENT, AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL NORDIC SEMICONDUCTOR ASA OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+ * TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+ * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+export const bleChannels = [
+    37, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 38, 11, 12, 13, 14, 15, 16, 17, 18,
+    19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 39,
+];
+
+export const bleChannelRange = {
+    lower: Math.min(...bleChannels),
+    upper: Math.max(...bleChannels),
+};
```

Comment:
How do you plan to use this? For me _lower_ and _upper_ sounds like a function, however their values are the min and max.
You also could just assign the min and the max to the array itself:
```js
bleChannels.min = 0;
bleChannels.max = 39;
```

Reply:
I am currently using this in two places: In the RSSI app in [the reducer]( https://github.com/NordicSemiconductor/pc-nrfconnect-rssi/blob/7a10e34a5b04144df04bdf20cd10d93ce54aea3d/src/reducer.js#L52) and in [the component to select the displayed channel range](https://github.com/NordicSemiconductor/pc-nrfconnect-rssi/blob/7a10e34a5b04144df04bdf20cd10d93ce54aea3d/src/SidePanel/ChannelRange.jsx#L60-L73).

I also struggle with the names. To me `min` and `max` more sound like functions (maybe because of `Math.min/max`) so I started using `lower` and `upper` in several places but also thought about `start` and `end` occasionally. Maybe I will think about that again and see whether it looks better when I depart from `lower/upper` in the RSSI app.

Usually I avoid putting additional properties on Arrays, being afraid that they mess up other code unexpectedly. But I could use `Object.defineProperty` which would at least not make `min` and `max` (and `isAdvertisementChannel`) enumerable properties. I like that idea and will do so in another PR.